### PR TITLE
Pkg/Template restructure

### LINKED ISSUE #1087: Moved CLI commands to consolidate packages under moniker templates
These changes are to remove the mention of packages from our public CLI interface. The change incorporates many conventions from popular CLIs in infrastructure (kubectl for instance).

The old command still work, but they are hidden in the CLI. It'd benefit us to remove them from our docs imo, but that is a decision for you all. I'd like to axe those deprecated commands within the next ~month.

##### Relevant URLs
- basically all the pkg docs 😬 
- PR driving changes: https://github.com/influxdata/influxdb/pull/18392

### LINKED ISSUE #1088: Restructure pkg and stack CLI docs
PR: https://github.com/influxdata/influxdb/pull/18392

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/cli/influx/
- https://v2.docs.influxdata.com/v2.0/reference/cli/influx/pkg
- https://v2.docs.influxdata.com/v2.0/reference/cli/influx/pkg/stack
- and children pages

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1087
Closes #1088 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/influxdb-templates/use.md [hunk 1]
```diff
@@ -89,99 +91,99 @@ Validate a template stored in your local filesystem or from a URL.
...
-To install all templates in a directory, use the `-f` or `--file` flag to provide
+### Apply all templates in a directory
+To apply all templates in a directory, use the `-f` or `--file` flag to provide
 the **directory path** of the directory where template manifests are stored.
-By default, this only installs templates stored in the specified directory.
-To install all templates stored in the specified directory and its subdirectories,
-include the `--recurse` flag.
+By default, this only applies templates stored in the specified directory.
+To apply all templates stored in the specified directory and its subdirectories,
+include the `-R`, `--recurse` flag.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/influxdb-templates/use.md:145]: Yeah, they introduces a shorthand version of the flag, `-R`.

File: content/v2.0/reference/cli/influx/stacks/_index.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
 
-The `influx pkg stack list` command lists InfluxDB stacks and associated resources.
+The `influx stacks` command and its subcommands list and manage InfluxDB stacks
+and associated resources.
 
 ## Usage
 ```
-influx pkg stack list [flags]
+influx stacks [flags]
+influx stacks [command]
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/reference/cli/influx/stacks/_index.md:22]: This is how the CLI itself outputs the usage and is the same throughout the CLI docs. I think the argument is that using flags is more common than using subcommands.

File: content/v2.0/influxdb-templates/use.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsteenb2) [reply content/v2.0/influxdb-templates/use.md:145]: the `-R` is a familiar flag for recursing into  a directory

File: content/v2.0/reference/cli/influx/apply/_index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+_For information about finding and using InfluxDB templates, see
+[Use InfluxDB templates](/v2.0/influxdb-templates/use/)._
+
+## Usage
+```
+influx apply [flags]
+```
+
+#### Aliases
+`apply`, `pkg`
```
Comment (jsteenb2) [inline content/v2.0/reference/cli/influx/apply/_index.md:24]: would it be possible to mark the `pkg` alias as deprecated?

File: content/v2.0/reference/cli/influx/stacks/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsteenb2) [reply content/v2.0/reference/cli/influx/stacks/_index.md:22]: That's a great question, I'm not sure why the output is like that. It is generated by the library we use to create the CLI 🤔

File: content/v2.0/reference/cli/influx/stacks/init.md [hunk 4]
```diff
@@ -24,8 +26,17 @@ influx pkg stack init [flags]
 | `--json`                    | Output data as JSON (default `false`) |                 | `INFLUX_OUTPUT_JSON`  |
 | `-o`, `--org`               | Organization name                     | string          | `INFLUX_ORG`          |
 | `--org-id`                  | Organization ID                       | string          | `INFLUX_ORG_ID`       |
-| `-u`, `--package-url`       | Package URLs to associate stack       | list of strings |                       |
 | `-d`, `--stack-description` | Stack description                     | string          |                       |
 | `-n`, `--stack-name`        | Stack name                            | string          |                       |
+| `-u`, `--template-url`      | Package URLs to associate stack       | list of strings |                       |
```
[the remark this answers is not in the document]
Comment (jsteenb2) [reply content/v2.0/reference/cli/influx/stacks/init.md:31]: great catch! I'll remedy this in the CLI now 👍. It should be `Tempalte URLs`

[the remark this answers is not in the document]
Comment (jsteenb2) [reply content/v2.0/reference/cli/influx/stacks/init.md:31]: nvm, the CLi has it as `Template URLs`

File: content/v2.0/reference/cli/influx/apply/_index.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/apply/_index.md:24]: Done.

**Pull request merged**